### PR TITLE
The quantum algorithm zoo link was not working; added two new courses.

### PR DESCRIPTION
**Pull request by SSdevelop**
List of changes made:
1. The [NIST link](https://math.nist.gov/quantum/zoo/) for the quantum algorithm zoo was not working. Added the new link which works and provides a list of quantum algorithms. 
2. Added two courses. One of them is an introductory course from the Saint Petersburg State University which focuses more on concepts. The other is a MicroMaster programme by Purdue University focusing on quantum hardware, sensors, and algorithms.

### CODE REVIEW

File: qosf.org/learn_quantum.md [hunk 1]
```diff
@@ -105,6 +105,12 @@ A web class for _high school students_. Organized by [QuSoft](http://www.qusoft.
 #### [Quantum Machine Learning](https://www.edx.org/course/quantum-machine-learning)
 Hands-on introduction to quantum computing and quantum-enhanced machine learning, complete with a [code repo](https://gitlab.com/qosf/qml-mooc/).
 
+#### [Quantum Computing. Less Formulas - More Understanding](https://www.coursera.org/learn/quantum-computing-lfmu)
+An introductory course on Quantum Computing, focusing more on concepts.
+
+#### [Quantum Technology: Detectors and Networking](https://www.edx.org/micromasters/purduex-quantum-technology-detectors-and-networking?index=product&queryID=b3d92b1eebc019dc082bb40feca0f978&position=3)
+An online program from Purdue University teaching the funcamentals of quantum networking,
```
Comment (tbabej) [inline qosf.org/learn_quantum.md:112]:
```suggestion
An online program from Purdue University teaching the fundamentals of quantum networking,
```

File: qosf.org/learn_quantum.md [hunk 2]
```diff
@@ -105,6 +105,12 @@ A web class for _high school students_. Organized by [QuSoft](http://www.qusoft.
...
+detectors and sensors and applied quantum computing.
```
Comment (tbabej) [inline qosf.org/learn_quantum.md:113]:
```suggestion
detectors, sensors and applied quantum computing. 
```

**Review by tbabej — CHANGES_REQUESTED** [review]
Looks good! I left two minor comments on the description of the Purdue course.

File: qosf.org/learn_quantum.md (hunk 1 above)
Comment (SSdevelop) [reply qosf.org/learn_quantum.md:112]: Thank you for the fix.

File: qosf.org/learn_quantum.md (hunk 2 above)
Comment (SSdevelop) [reply qosf.org/learn_quantum.md:113]: Thank you for the fix. I will commit the suggestions.

**Review by tbabej — APPROVED** [review]

**Pull request merged**